forem/forem
# Feature:Notify Devs when Sidekiq is Not Running in Development

## What type of PR is this? (check all applicable)

## Description
People missing Elasticsearch data seems to be a common issue in development which got me thinking about how we could alert people of an issue. However, usually missing Elasticsearch data is linked to Sidekiq not running which means things are not getting indexed. Rather than trying to compare ES and db counts I decided to add a warning for development if Sidekiq is not running. 

I am not a huge fan of having development specific code but I thought this would be nice to have. Let me know what you all think!

![Screen Shot 2020-10-02 at 5 32 09 PM](https://user-images.githubusercontent.com/1813380/94974798-4198a500-04d5-11eb-83c1-336bdc937412.png)


## QA Instructions, Screenshots, Recordings
1. Boot app with sidekiq running and see that no banner is displayed
2. boot app without sidekiq, `rails s` and verify the banner is displayed. 

## Added tests?
Since this is just for development I thought leaving it without tests is fine. Plus the other validation setup methods also did not have specs. 


![alt_text](https://media1.tenor.com/images/c3611aecb0f40180ad67a64f6efe2927/tenor.gif?itemid=15079115)
